src/conf.cpp

```diff
@@ -20,10 +22,13 @@ Config::Config() {
     this->relative_timestamps = 0;
     this->pcap = 0;
     this->csv = 0;
+    this->wireless = 0;
```

Comment:
Same comment as before RE mixed traffic and if it's not just better to parse this.

Reply:
I didn't find a good way to distinguish the eth header from radiotap header by simply parsing some bits. Because for eth header, it's just src_mac + dst_nac + type for the next header. I tried to search for it, but haven't found a good keyword. Will "Linux Cooked header types" help in this parsing?

Reply:
Anything we can use in here? https://linux.die.net/man/7/pcap-linktype

Reply:
I just tried it. It works, and can easily distinguish the radiotap header from ethernet header. This library is magic! I will refine my code accordingly.

Reply:
Resolved